bin/test_staging_update_census_mapbox

```diff
@@ -1,17 +1,26 @@
 #!/usr/bin/env ruby
 
 # The map on code.org/yourschool is run off of Mapbox.
-# This script updates the data on Mapbox by querying all rows from
-# census_summaries table for the appropriate school year,
-# creating maptiles with Mapbox's command line tool, Tippecanoe,
-# and uploading them to Mapbox's server.
+# This script updates the data on Mapbox by reading the rows from
+# the given AWS file and creating maptiles with Mapbox's command
+# line tool, Tippecanoe, and uploading them to Mapbox's server
+# under the `censustiles-dev` tileset.
+#
+# This can be used on staging to test that the data and script will
+# work with the new year's data before merging + running on
+# production using ./bin/update_census_mapbox.
+#
+# Note: This script does not read from the CensusSummaries table while
+# the real update_census_mapbox script does. Both read from the
+# Schools database.
 
 require_relative '../dashboard/config/environment'
 require_relative '../pegasus/helpers/properties'
-
 require_relative './cron/upload_to_mapbox'
 
-raise "CDO.mapbox_upload_token is not defined" unless CDO.mapbox_upload_token
```

Comment:
Curious why this was removed?

Reply:
Oh thanks for catching this, I kept confusing `mapbox_upload_token` with `mapbox_upload_tileset`. I'll fix this!